docs/fsharp-cheatsheet.md

```diff
@@ -555,136 +555,158 @@ Another way of implementing interfaces is to use *object expressions*.
 <a name="CodeOrganization_Modules"></a>
 ### Modules
 Modules are key building blocks for grouping related code; they can contain `types`, `let` bindings, or (nested) sub `module`s.
-Identifiers within modules can be referenced using dot notation, or you can bring them into scope via the `open` keyword. Illustrative-only example:
-
-    module Game =
-        let mutable basePoints = 1
-        type Player = { id: int; score: int }
-        let playerScored player = { player with score = player.score + basePoints }
-
-    let player: Game.Player = { id = 1; score = 0 }
-    let player' = Game.playerScored player  // score = 1
-
-    open Game
-    basePoints <- 2
-    let player'' = playerScored player'  // player''.score = 3
-
-If you have only one module in a file, the `module` name can be declared at the top of the file, with all further declarations
-being module elements (and non indentation required)
+Identifiers within modules can be referenced using dot notation, or you can bring them into scope via the [`open`](#CodeOrganization_OpenAndAutoOpen) keyword.
+Illustrative-only example:
+
+    module Money =
+        type CardInfo =
+            { number: string
```

Comment:
do you indent by 2 or 4?

Reply:
I don't really have a preference, I use the defaults, which I guess is 4 here.